daemon/config/config_test.go

```diff
@@ -217,105 +225,88 @@ func TestFindConfigurationConflictsWithMergedValues(t *testing.T) {
 }
 
 func TestValidateConfigurationErrors(t *testing.T) {
+
+	config0, _ := New()
+	config0.CommonConfig.Labels = []string{"one"}
+	config00, _ := New()
+	config00.CommonConfig.Labels = []string{"foo=bar", "one"}
+	config1, _ := New()
+	config1.CommonConfig.DNSConfig.DNSSearch = []string{"123456"}
+	config11, _ := New()
+	config11.CommonConfig.DNSConfig.DNSSearch = []string{"a.b.c", "123456"}
+	config2, _ := New()
+	config2.CommonConfig.BridgeConfig.DefaultBridgeConfig.MTU = -10
+	config3, _ := New()
+	config3.CommonConfig.MaxConcurrentDownloads = -10
+	config4, _ := New()
+	config4.CommonConfig.MaxConcurrentUploads = -10
+	config5, _ := New()
+	config5.CommonConfig.MaxDownloadAttempts = -10
+	config55, _ := New()
+	config55.CommonConfig.MaxDownloadAttempts = 0
+	config6, _ := New()
+	config6.CommonConfig.NodeGenericResources = []string{"foo=bar", "foo=baz"}
+	config7, _ := New()
+	config7.CommonConfig.NodeGenericResources = []string{"foo=1"}
+	config8, _ := New()
+	config8.CommonConfig.Hosts = []string{"tcp://127.0.0.1:2375"}
+	config9, _ := New()
+	config9.CommonConfig.LogLevel = "warn"
```

Comment:
Why is this change necessary? It looks like the test implementation uses mergo to merge the partial config in the table on top of defaults, making this change redundant.